zuul.yaml

```diff
@@ -0,0 +1,4 @@
+---
+- project:
+    templates:
+      - ansible-collections-trendmicro-deepsec
```

Comment:
This will work if the template exists, If you just want to run some jobs, you can also do:

```
---

- project:
    check:
      jobs:
        - name-of-my-job
```